### PR TITLE
Add Medik8s icon for MDR

### PR DESCRIPTION
**Pull request by razo7**
Adding an icon for MDR is needed, since bundle validation in #55  fails (while #56 only raised a warning on bundle validation).
ATM we use [Medik8s icon](https://github.com/medik8s/docs/blob/gh-pages/images/medik8s-logo.png), since it is only the first release of FAR and we might change it in the future.

We do provide two icon versions for better identification between _upstream_ and _downstream_ operators - we use **blue** icons upstream and **red** icons for downstream.

Similar PR to FAR's PR -> https://github.com/medik8s/fence-agents-remediation/pull/37

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -267,10 +267,14 @@ bundle: manifests kustomize operator-sdk ## Generate bundle manifests and metada
...
 .PHONY: bundle-update
+DEFAULT_ICON_BASE64 := $(shell base64 --wrap=0 ./config/assets/medik8s_blue_icon.png)
+export ICON_BASE64 ?= ${DEFAULT_ICON_BASE64}
 export CSV="./bundle/manifests/$(OPERATOR_NAME).clusterserviceversion.yaml"
-bundle-update: ## Update containerImage and createdAt
+
+bundle-update: ## Update containerImage, createdAt, icon, and displayName fields in the bundle's CSV, then validate the bundle directory
 	sed -r -i "s|containerImage: .*|containerImage: $(IMG)|;" ${CSV}
 	sed -r -i "s|createdAt: .*|createdAt: \"`date '+%Y-%m-%d %T'`\"|;" ${CSV}
+	sed -r -i "s|base64data:.*|base64data: ${ICON_BASE64}|;" ${BUNDLE_CSV}
```
Comment (clobrano) [inline Makefile:277]: this seems a typo `s/BUNDLE_CSV/CSV`
Otherwise it is ok to use `BUNDLE_CSV` everywhere, if you think it is more descriptive

**Review by clobrano — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (razo7) [reply Makefile:277]: That's indeed a mistake.
Basically there are two CSV in the code, one under `config` and one under `bundle`, and that's why I thought of mentioning it. But we are under a section of  `Bundle Creation Addition` so it should be clear that the CSV is of bundle. 
So what do you prefer `BUNDLE_CSV` or just `CSV`?

Comment (clobrano) [reply Makefile:277]: The variable is defined few lines above, I think `CVS` is clear enough

**Pull request merged**